jupyter-lib/api/src/main/kotlin/org/jetbrains/kotlinx/jupyter/api/renderersHandling.kt

```diff
@@ -137,7 +137,7 @@ class SubtypeRendererTypeHandler(private val superType: KClass<*>, override val
     override fun acceptsType(type: KClass<*>): Boolean {
         return try {
             type.isSubclassOf(superType)
-        } catch (e: UnsupportedOperationException) {
+        } catch (e: Throwable) {
```

Comment:
Let's just catch `KotlinReflectionInternalError` additionally? I will also file the issue in Kotlin, because there is the following comment above the line that throws this exception:
```
// Should not happen since a proper Kotlin- or Java-class must have been resolved
throw KotlinReflectionInternalError("Unresolved class: $jClass")
```

Reply:
https://youtrack.jetbrains.com/issue/KT-48636